docs/usage.md

```diff
@@ -97,6 +97,13 @@ run_topostats
 ...
 ```
 
+On a successful completion you should see a message similar to the following which indicates various aspects of the run
+along with information about how to give feedback, report bugs and cite the software.
+
+``` bash
+
```

Comment:
Forgot to add example output statement here, or perhaps this was meant to be removed, since you give an example completion message on lines `191` to `214`.